python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -206,7 +207,7 @@ def setup_operator(self, training_models):
             self.dist_backend = TorchDistBackend()
 
     def train_epochs(self, data_creator, epochs=1, batch_size=32, profile=False,
-                     info=None, wrap_dataloader=None, callbacks=None,
+                     info=None, wrap_dataloader=None, callbacks=[],
```

Comment:
seems in Python, using a mutable list as default value is not recommended?
Still keep it None and change to an empty list in the function.